<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>animation</class>
 <widget class="QWidget" name="animation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>animation</string>
  </property>
  <widget class="QPushButton" name="scale_btn">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>1</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>缩放scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displacement_btn">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>26</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>位移</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>207</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>透明度opacity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="slider_text"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
